scripts/run-e2e-tests-on-server

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+# Expect first parameter to be packit base url and second parameter to be auth method
+if [[ "$1" =~ ^https?://([^/]+) ]]; then
+    BASE_URL=$1
+else
+    echo "Invalid Packit server URL: $1"
+    exit 1
+fi
+
+if [[ $2 != "basic" || $2 != "github" ]]; then
+  AUTH_METHOD=$2
+else
+  echo "Unknown auth method: $2"
+  exit 1
+fi
+
+export PACKIT_E2E_BASE_URL=$BASE_URL
+export PACKIT_E2E_AUTH_METHOD=$AUTH_METHOD
+
+echo "Server is $BASE_URL. Auth method is $AUTH_METHOD."
+
+if [[ $AUTH_METHOD == "basic" ]]; then
+  echo "Please provide user credentials for running e2e tests. User will need to have admin permissions in Packit."
+  read -p 'Username: ' BASIC_USER
+  read -sp 'Password (will be hidden):' BASIC_PASSWORD
+  echo
```

Comment:
Is this necessary, especially given that `auth.setup.ts` applies some defaults to it?

Reply:
Only if we had a non-localhost basic auth deployment, which we don't at the moment but  we could do!